llvm-general/src/LLVM/General/Internal/Attribute.hs

```diff
@@ -50,6 +51,12 @@ instance Monad m => EncodeM m A.PA.ParameterAttribute (Ptr FFI.ParameterAttrBuil
       A.PA.InAlloca -> FFI.parameterAttributeKindInAlloca
       A.PA.NonNull -> FFI.parameterAttributeKindNonNull
       A.PA.Returned -> FFI.parameterAttributeKindReturned
+      A.PA.SwiftSelf -> FFI.parameterAttributeKindSwiftSelf
+      A.PA.SwiftError -> FFI.parameterAttributeKindSwiftError
+      A.PA.WriteOnly -> FFI.parameterAttributeKindWriteOnly
+      A.PA.Alignment _ -> error "Impossible"
+      A.PA.Dereferenceable _ -> error "Impossible"
+      A.PA.DereferenceableOrNull _ -> error "Impossible"
```

Comment:
If you're going spend the code space on these cases, please make their error messages actually provide useful information as to how to fix the presumed problem, like "llvm-general internal error: cases inconsistent in ParameterAttribute encoding for " ++ show a